doc/unet.txt

```diff
@@ -0,0 +1,161 @@
+.. _unet:
+
+U-Net
+**********************************************
+
+.. note::
+    This section assumes the reader has already read through :doc:`lenet` for
+    convolutional networks motivation and :doc:`fcn_2D_segm` for segmentation
+    network.
+
+Summary
++++++++
+
+This tutorial provides a brief explanation of the U-Net architecture as well as a way to implement
+it using Theano and Lasagne. U-Net is a Fully Convolutional Network (FCN) that does image segmentation.
+Its goal is then to predict each pixel's class. See :doc:`fcn_2D_segm` for differences between
+network architecture for classification and segmentation tasks.
+
+Data
+++++
+
+The data is from ISBI challenge and can be found `here <http://brainiac2.mit.edu/isbi_challenge/home>`_.
+
```

Comment:
Also, there seems to be data augmentation by default during training.
- it would be nice to mention that, maybe a sentence or two about what that does and why it matters
- also, it uses another dependency, SimpleITK. We should indicate it is an additional dependency.